prompt/prompt-llm/src/commonMain/kotlin/ai/koog/prompt/llm/OllamaModels.kt

```diff
@@ -5,12 +5,64 @@ package ai.koog.prompt.llm
  * Each maker contains specific models with configurations such as unique identifiers and capabilities.
  */
 public object OllamaModels {
+    /**
+     *  The Groq object represents the configuration for the Groq large language model (LLM).
+     *  It contains the predefined model specifications for Groq's LLMs, including their identifiers
+     *  and supported capabilities.
+     */
+    public object Groq {
+        /**
+         * Represents the LLAMA version 3.0 model provided by Groq.
+         *
+         * This variable defines an instance of the `LLModel` class with the Ollama provider, a unique identifier "llama3",
+         * and a set of capabilities. The supported capabilities include:
+         *  - Temperature adjustment.
+         *  - JSON Schema-based tasks (Not very clear).
+         *  - Tool utilization.
+         *
+         * LLAMA 3 is designed to support these specified features, enabling developers to utilize the model for tasks
+         * that require dynamic behavior adjustments, schema adherence, and tool-based interactions.
+         */
+        public val LLAMA_3_GROK_TOOL_USE: LLModel = LLModel(
+            provider = LLMProvider.Ollama,
+            id = "llama3-groq-tool-use:latest",
```

Comment:
`latest` is a tag, not particular model. As far as I see - there are two models - `llama3-groq-tool-use:8b`, `llama3-groq-tool-use:70b`, and 8b is latest currently. I would prefer we have two cards for these particular models.